common/discord/discord.cpp

```diff
@@ -0,0 +1,92 @@
+#include "discord.h"
+#include "../http/httplib.h"
+#include "../json/json.h"
+#include "../string_util.h"
+#include "../eqemu_logsys.h"
+
+constexpr int MAX_RETRIES = 10;
+
+void Discord::SendWebhookMessage(const std::string &message, const std::string &webhook_url)
+{
+	// validate
+	if (webhook_url.empty()) {
+		LogDiscord("[webhook_url] [{}] is empty", webhook_url);
```

Comment:
No reason to log `webhook_url` itself if it's empty (unless this is an intentional log output thing)